cli/rt/02_console.js

```diff
@@ -462,7 +462,11 @@
       .replace(/\n/g, "\\n")
       .replace(/\r/g, "\\r")
       .replace(/\t/g, "\\t")
-      .replace(/\v/g, "\\v");
+      .replace(/\v/g, "\\v")
+      .replace(
+        /[\x00-\x1f\x7f-\x99]/g,
```

Comment:
```suggestion
        /[\x00-\x1f\x7f-\x9f]/g,
```

It looks like this should extend to `\x9f`: https://en.wikipedia.org/wiki/List_of_Unicode_characters#Control_codes